core/types/transaction_meta.go

```diff
@@ -13,6 +13,15 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 )
 
+type QueueOrigin int64
+
+const (
+	// Possible `queue_origin` values
+	QueueOriginL1ToL2    QueueOrigin = 0
+	QueueOriginSafety    QueueOrigin = 1
```

Comment:
@karlfloersch I believe the version of the contracts in geth have the concept of a safety queue, so leaving this for now.

Reply:
Yep!